docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+
+After adding this code, run
+
+    npm start
+
+during development to automatically compile all JSX and Javascript syntax into one `bundle.js` which you can then directly link to your `index.html` file. Use
+
+    npm run build
+
+to compile a production ready bundle of your code.
+
+## TodoCreateForm.jsx
+Let's start by creating the top level Todo component using `Tux's` `createOwnerClass`. The Owner class is also responsible for the state of our Todo application. In `Tux` you are able to easily connect your Owner components to their stores using the `connectOwnerToStore` method which allows you to designate static properties and methods you want to automatically expose child components on the key of `nearestOwnerProps`. As the top-level Todo component it is also be responsible for exposing its methods and properties its children components. This is accomplished by using the `registerOwnerProps` method, which allows you to specify which properties and methods you want to expose to children components. Note these properties and methods should be static, if you need to pass down mutable properties, use the standard `React` practices to flow down `props`.
+
+**NOTE** We are going to build out the `todoStore`, `todoActions`, and the `TodoCreateForm` later in the Getting Started Guide.
+
+```javascript
+    var React = require('tux/React');
+    var todoActions = require('./todoActions');
+    var todoStore = require('./todoStore');
+    var Todo = require('./Todo.jsx');
+    var TodoCreateForm = require('./TodoCreateForm.jsx');
+
+    var TodoViewOwner = React.createOwnerClass({
+      getInitialState: function () {
+        return {
+          todos: todoStore.getAll() // Will get all of the Todos that are stored in the TodoStore
+        };
+      },
+
+      // This method connects this owner component to the specified store. Additionally it also adds and removes the change listeners from the store during the appropriate lifecycle events, specifically componentDidMount and componentWillUnmount.
+      connectOwnerToStore: function () {
+        return {
+          store: todoStore,
+          listener: function () {
+            this.setState({ todos: todoStore.getAll() });
+          }.bind(this)
+        };
+      },
+
+      // The registerOwnerProps method is invoked with the context of the component and allows an OwnerClass to automatically expose its properties to its child components (direct or otherwise). In this instance this component is exposing the add and remove methods specified in the return.
+      registerOwnerProps: function () {
+        // the todoActions.METHOD will take the input to that method and pass it to the dispatcher, where it will broadcast the METHOD to any store listening. If that action is registered with the store, the payload from that action will then be passed to the corresponding actions method.
```

Comment:
Shorten explanation: These methods will dispatch the corresponding actions to any listening stores. Additionally, you can pass in an object and the store will receive it as an input to its callback.